allennlp/data/tokenizers/word_filter.py

```diff
@@ -74,33 +75,34 @@ def __init__(self,
-                                  'aside', 'astride', 'at', 'atop', 'back', 'be', 'because', 'before',
-                                  'behind', 'below', 'beneath', 'beside', 'besides', 'between',
-                                  'beyond', 'but', 'by', 'concerning', 'do', 'down', 'due', 'during',
-                                  'either', 'except', 'exclusive', 'false', 'for', 'from', 'happen',
-                                  'he', 'her', 'hers', 'herself', 'him', 'himself', 'his', 'how',
-                                  'how many', 'how much', 'i', 'if', 'in', 'including', 'inside',
-                                  'instead', 'into', 'irrespective', 'is', 'it', 'its', 'itself',
-                                  'less', 'me', 'mine', 'minus', 'my', 'myself', 'neither', 'next',
-                                  'not', 'occur', 'of', 'off', 'on', 'onto', 'opposite', 'or', 'our',
-                                  'ours', 'ourselves', 'out', 'out of', 'outside', 'over', 'owing',
-                                  'per', 'prepatory', 'previous', 'prior', 'pursuant', 'regarding',
-                                  's', 'sans', 'she', 'subsequent', 'such', 'than', 'thanks', 'that',
-                                  'the', 'their', 'theirs', 'them', 'themselves', 'then', 'these',
-                                  'they', 'this', 'those', 'through', 'throughout', 'thru', 'till',
-                                  'to', 'together', 'top', 'toward', 'towards', 'true', 'under',
-                                  'underneath', 'unlike', 'until', 'up', 'upon', 'us', 'using',
-                                  'versus', 'via', 'was', 'we', 'were', 'what', 'when', 'where',
-                                  'which', 'who', 'why', 'will', 'with', 'within', 'without', 'you',
-                                  'your', 'yours', 'yourself', 'yourselves', ",", '.', ':', '!', ';',
-                                  "'", '"', '&', '$', '#', '@', '(', ')', '?'])
+            self.stopwords = set([token.lower() for token in
+                                  ['I', 'a', 'aboard', 'about', 'above', 'accordance', 'according',
+                                   'across', 'after', 'against', 'along', 'alongside', 'also', 'am',
+                                   'amid', 'amidst', 'an', 'and', 'apart', 'are', 'around', 'as',
+                                   'aside', 'astride', 'at', 'atop', 'back', 'be', 'because', 'before',
+                                   'behind', 'below', 'beneath', 'beside', 'besides', 'between',
+                                   'beyond', 'but', 'by', 'concerning', 'do', 'down', 'due', 'during',
+                                   'either', 'except', 'exclusive', 'false', 'for', 'from', 'happen',
+                                   'he', 'her', 'hers', 'herself', 'him', 'himself', 'his', 'how',
+                                   'how many', 'how much', 'i', 'if', 'in', 'including', 'inside',
```

Comment:
Interestingly lowercase `'i'` already appears here. Any chance this filter was intended to be case sensitive originally and the `.lower()` call in `filter_words` is the bug?

Reply:
Not sure, but lowercasing seems correct in the sense that I would expect stopwords which were capitalized to be filtered, so I think i'll leave it as is.